db/migrate/20150912205745_create_seed_trades.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSeedTrades < ActiveRecord::Migration
+  def change
+    create_table :seed_trades do |t|
+      t.text       :message
+      t.text       :address
+      t.datetime   :requested_date
+      t.datetime   :accepted_date
+      t.datetime   :declined_date
```

Comment:
Hello cesy, distinction between accepted_date and declined_date are here for some reasons:

1- Column 'status' in the seed_trade#index related view shows the status of the trade request.

2- Seed owner can only mark as sent once he has accepted the trade.

I think this is most in accordance with Skud's issue post.